common/src/main/java/io/netty/util/AsyncMapping.java

```diff
@@ -18,11 +18,11 @@
 import io.netty.util.concurrent.Future;
 import io.netty.util.concurrent.Promise;
 
-public interface AsyncMapping<IN, OUT> {
+public interface AsyncMapping<U, R> {
```

Comment:
Does `U` mean anything, or is it an alternative when `T` is already taken? `P` for parameter type, and `R` for return type would make more sense to me than `U`. However I'm not sure it is worth moving away `IN` and `OUT` which seems more descriptive. Can we handle this as a followup PR?